src/pages/index.js

```diff
@@ -70,25 +80,65 @@ cardAddButton.addEventListener("click", () => {
 });
 
 /* Profile Edit Functions */
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const userInfo = new UserInfo({});
```

Comment:
`const userInfo = new UserInfo({});
`

* According to the task `UserInfo` should accept 3 **selectors** to search 3 elements: `this.#nameElement`, `this.#descriptionElement` and `this.#avatarElement `